### ISSUE TITLE
chezmoi needs a logo

### ISSUE THREAD

**Issue by twpayne** [twpayne/chezmoi#308]
chezmoi needs a logo, both for the homepage, and for app stores like snap.

Suggested keywords: computer, home, security, multiple

**Comment by mirzazulfan** [comment]
Hi @twpayne What do you think about this? if you need something else please let me know. Regard!
![chezmoi](https://user-images.githubusercontent.com/36124339/59253895-60788580-8c59-11e9-804b-c0e3d7aee6c0.png)

**Comment by twpayne** [comment]
Thanks very much for this @mirzazulfan :)

Could I ask for a few specific changes, specifically focus on the house/key icon, drop the monitors and the "chezmoi" text, something like:

```
      /   \
    /       \
  /           \
/__     O     __\
   |   /_\  |
   |        |
   |________|
```

Thanks!

**Comment by mirzazulfan** [comment]
Thanks for your response, How about this? if it's still not suitable, please let me know. Regard!
![CHEZ](https://user-images.githubusercontent.com/36124339/59405863-37c9ca80-8dd6-11e9-9ce8-cfc408560478.png)

**Comment by twpayne** [comment]
I really like number 3 :) Let's go with this with a couple of minor tweaks:

* Please shift the visual weight of the arc to the left so it aligns vertically with the visual weight of the house and the keyhole.
* Please make the left and right ends of the arc align with ends of the house roof.
* Please make the walls of the house slightly taller so the overall icon fits in a square and feels "square".
* Please remove the "chezmoi" text.

Would you be able to submit this as a PR with the icon in SVG or other open vector format?

Many thanks - this is really nice.

**Comment by mirzazulfan** [comment]
I'm glad you liked it. Do you mean like this? I'll send you .svg and other format.
![chezzmoi](https://user-images.githubusercontent.com/36124339/59443258-d7627980-8e25-11e9-8720-30eb658ca46e.png)

**Comment by twpayne** [comment]
Number 3 is perfect, thank you!

**Comment by mirzazulfan** [comment]
I've created a pr  #348 . . If you need to change in the future, please let me know. Best regard!

**Comment by twpayne** [comment]
Fixed thanks to @mirzazulfan in #348.
